docker-compose.sc.yml

```diff
@@ -0,0 +1,70 @@
+version: '2.1'
+services:
+  rubyrails:
+    container_name: "classroom_rubyrails"
+    build: 
+      context: .
+      dockerfile: Dockerfile-sc
+    volumes: 
+      - .:/usr/src/app
```

Comment:
.bundle and vendor are copied into the docker image, then not mounted back onto the host box. That way dependencies we shouldn't be editing don't clutter up the filesystem.